Makefile

```diff
@@ -126,18 +126,6 @@ build:  ## Build ingress controller, debug tool and pre-stop hook.
 		build/build.sh
 
 
-.PHONY: build-plugin
```

Comment:
Can we move the make target to us go release instead of deleting it altogether

Reply:
Seems like that'd require 2 make targets in the end, as goreleaser will refuse to run locally without specifying `--snapshot` and we'd need one target for production releases that doesn't include `--snapshot`.